docs/releases/1.7.txt

```diff
@@ -637,7 +637,7 @@ Forms
   value.
 
 * :attr:`SelectDateWidget.months
-  <django.forms.extras.widgets.SelectDateWidget.months>` can be used to
+  <django.forms.SelectDateWidget.months>` can be used to
```

Comment:
I'm pretty sure this would create a broken ref. Unless we have some special-casing for the releases directory but I don't think we do.